src/app/components/PageLayout/NetworkButton.tsx

```diff
@@ -115,11 +119,14 @@ export const StyledMobileNetworkButton = styled(Button)(({ theme }) => ({
 
 export const MobileNetworkButton: FC<NetworkButtonProps> = ({ isOutOfDate, layer, network, onClick }) => {
   const { t } = useTranslation()
-  const labels = getLayerLabels(t)
+
+  const label = hasFixedLayer() // If we are fixed to a layer,
+    ? getNetworkNames(t)[network] // let's show the name of the network,
+    : getLayerLabels(t)[layer] // otherwise, the name of the layer.
```

Comment:
This could be some utility, it is duplicted twice in this file.

Reply:
OK, created the `getNetworkButtonLabel()` function.